package/yast2-registration.spec

```diff
@@ -58,6 +58,7 @@ BuildRequires:  yast2-slp >= 3.1.9
 # packager/product_patterns.rb
 BuildRequires:  yast2-packager >= 3.1.95
 BuildRequires:  yast2-update >= 3.1.36
+BuildRequires:	autoyast2-installation
```

Comment:
With 
```ruby
stub_const("Yast::AutoinstConfig", double(selected_product: ...))
```
we can avoid also this build dependency.

Reply:
hm, yes. I'll do so.